clang/test/Preprocessor/sycl-macro.cpp

```diff
@@ -32,8 +35,10 @@
 // CHECK-NO-SYCL_FIT_IN_INT-NOT:#define __SYCL_ID_QUERIES_FIT_IN_INT__ 1
 // CHECK-SYCL-ID:#define __SYCL_ID_QUERIES_FIT_IN_INT__ 1
 
-// CHECK-CUDA:#define __SYCL_CUDA_ARCH__ 800
-// CHECK-CUDA-NOT:#define __CUDA_ARCH__ 800
+// CHECK-CUDA:#define __SYCL_CUDA_ARCH__ [[ARCH_CODE]]
+// CHECK-CUDA-NOT:#define __CUDA_ARCH__ [[ARCH:[0-9]+]]
```

Comment:
I don't think we need to define `ARCH` (here or the next check) if we're not going to use it. We could just have `{{[0-9]+}}`

Reply:
Yeah makes sense. Updated.